invenio_files_rest/helpers.py

```diff
@@ -166,11 +181,14 @@ def send_stream(stream, filename, size, mtime, mimetype=None, restricted=True,
     # Set cache-control
     if not restricted:
         rv.cache_control.public = True
-        cache_timeout = current_app.get_send_file_max_age(filename)
+        cache_timeout = current_app.get_send_file_max_age(filename) or max_age
```

Comment:
Why the func is not enough?

Reply:
because that is a globally set default, i want to make it changeable also per function call, as is possible with the flask `send_file` function

Reply:
I just realized it should be the other way around then :smile: